### PR TITLE
feat: lab6 Git & GitHub

### PR DESCRIPTION
**Pull request by andy89923**

### CODE REVIEW

File: lab6/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Git & GitHub
```
Comment (ianchen0119) [inline lab6/README.md:1]:
```
# Lab 6: Git & GitHub
```

File: lab6/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+## What is Git/GitHub?
+
+> **ChatGTP**
+>
+> < What is Git? >
+>
+> >Git is a distributed version control system used to track changes in source code during software development. It allows multiple developers to work on a project simultaneously without interfering with each other’s changes.
+
+- [Git vs. GitHub: What's the difference?](https://www.youtube.com/watch?v=wpISo9TNjfU) -- from IBM
+
```
Comment (ianchen0119) [inline lab6/README.md:29]: Add this article for reference: https://blog.brachiosoft.com/posts/git/

File: lab6/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## Exercise
+
+- Fork https://github.com/andy89923/nf-example & Trace code 
+    - It's a sample of free5GC NFs 
+    - Same structure but no **consumer** 
+    - Not a real NF, for sure! 😆
+-  Follow the NF structure, create another API service
+    - Must include 1 GET and 1 POST method at least
```
Comment (ianchen0119) [inline lab6/README.md:80]: Should we add a test file here to test those APIs automatically?

Comment (andy89923) [reply lab6/README.md:80]: Sure, I think we can use colab colletcion in postman. 
(I will try to create one later).

Comment (ianchen0119) [reply lab6/README.md:82]: @andy89923 
Got it.
So, is this PR ready to merged (I'm not sure the collection file will put in this repo or other place)?

Comment (andy89923) [reply lab6/README.md:82]: Yes, this PR is ready!
If have any update, I will create another PR!
Thanks!

Postman can't work on different task and merge in GitHub(not sure), I will try on team in Postman.

Comment (ianchen0119) [reply lab6/README.md:82]: Hi @andy89923 
I think write the go test instead of postman may solve your problem.

Comment (andy89923) [reply lab6/README.md:82]: Ok, I will write in NF-Example.

I will update some instruction here after  the test is finished.

Comment (andy89923) [reply lab6/README.md:82]: @ianchen0119 Once everything is prepared, I will notify you.

**Review by ianchen0119 — APPROVED** [review]

**Pull request merged**